### DISCUSSION TITLE
MobTwins

### DISCUSSION THREAD

**Discussion by danechek** [PurpurMC/Purpur#523] [Feature Requests]
I suggest adding the chance of giving birth to twins to all non-hostile mobs, similar to [this plugin](https://www.spigotmc.org/resources/mobtwins.94665/).

**Comment by BillyGalbreath** [comment]
Good idea. I like it ^_^

**Reply by danechek** [reply]
> Good idea. I like it ^_^

I think it should look like this - a separate setting for each mob, as well as the ability to specify the chance in % for this. And maybe you will like the idea of ​​setting the chances of having twins and even triplets, because not only one or two can be born in real life)

**Comment by DereC4** [comment]
#1714
